.travis.yml

```diff
@@ -91,6 +91,16 @@ jobs:
       python: 3.6
       script: coverage run $(which nosetests) -c nose_unit.cfg
 
+    - stage: tests
+      name: "Python: 3.7 Unit Tests"
+      python: 3.7
+      script: coverage run $(which nosetests) -c nose_unit.cfg
```

Comment:
I think it's sufficient to test only 3.6 and 3.8. Hopefully it's going to be enough and will save us a lot of time.

Reply:
ok i've removed py 3.5 and 3.7 from appveyor and travis